File: Java/program/Shirt.java
//Kelas Shirt anak dari Clothing
class Shirt extends Clothing {
    //private atribut
    private String color;
    private String sleeve_type;
    
    //constructor
    public Shirt() {
    }
    
    public Shirt(String color, String sleeve_type) {
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    public Shirt(String idProduct, String name, String brand, int price, String size, String material, String gender,
            String color, String sleeve_type) {
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    //setter getter
    public String getColor() {
        return color;
    }
    public void setColor(String color) {
        this.color = color;
    }
    public String getSleeve_type() {
        return sleeve_type;
    }
    public void setSleeve_type(String sleeve_type) {
        this.sleeve_type = sleeve_type;
    }
}

File: Python/program/Shirt.py
#Kelas Shirt anak dari kelas clothing

from Clothing import Clothing

class Shirt(Clothing):
    __color = ""
    __sleeve_type = ""

    #constructor
    def __init__(self, idProduct, name, brand, price, size, material, gender, color, sleeve_type):
        super().__init__(idProduct, name, brand, price, size, material, gender)
        self.__color = color
        self.__sleeve_type = sleeve_type

    # Getter dan Setter untuk color
    def get_color(self):
        return self.__color

    def set_color(self, color):
        self.__color = color

    # Getter dan Setter untuk sleeve_type
    def get_sleeve_type(self):
        return self.__sleeve_type

    def set_sleeve_type(self, sleeve_type):
        self.__sleeve_type = sleeve_type